src/puppetlabs/puppetdb/scf/storage_utils.clj

```diff
@@ -223,6 +223,14 @@
   [column]
   (format "encode(%s::bytea, 'hex')" column))
 
+(defn vacuum-analyze
+  "Must call within a db connection"
+  [db]
+  (sql/with-db-connection [conn db]
+    (-> (doto (:connection conn) (.setAutoCommit true))
+        .createStatement
+        (.execute "vacuum (analyze)"))))
```

Comment:
There's a jdbc function for 231/232, http://clojure.github.io/java.jdbc/#clojure.java.jdbc/execute!, is this any different from what you have there?